sdk/core/azure-core/src/samples/java/com/azure/core/util/IterableStreamJavaDocCodeSnippets.java

```diff
@@ -43,9 +43,7 @@ public void streamSnippet() throws MalformedURLException {
             if (resp.getStatusCode() == HttpURLConnection.HTTP_OK) {
                 System.out.printf("Response headers are %s. Url %s%n", resp.getDeserializedHeaders(),
                     resp.getRequest().getUrl());
-                resp.getItems().forEach(value -> {
-                    System.out.printf("Response value is %d%n", value);
-                });
+                resp.getElements().forEach(value -> System.out.printf("Response value is %d%n", value));
```

Comment:
Will these change need corresponding readme.md file update? Or embedme already supports BEGIN/END tag?

Reply:
This is a change to the Javadoc codesnippets which will regenerate during javadoc:javadoc, changes to the README snippets without regenerating would cause a build failure in check-in